### ISSUE TITLE
quickstart deploy fails starting undercloud vm

### ISSUE THREAD

**Issue by pkilambi** [redhat-openstack/tripleo-quickstart#60]
I'm not sure if this is a bug or something specific to my env, but i keep hitting this consistently. When i try to run quickstart: it fails with "failed to retrieve file descriptor for interface: Transport endpoint is not connected"

TASK [setup/undercloud : Define undercloud vm] *********************************
Tuesday 22 March 2016  19:04:16 -0400 (0:00:21.933)       0:10:19.382 ********\* 
changed: [host0] => {"changed": true, "created": "undercloud"}

TASK [setup/undercloud : Start undercloud vm] **********************************
Tuesday 22 March 2016  19:04:16 -0400 (0:00:00.218)       0:10:19.600 ********\* 
fatal: [host0]: FAILED! => {"changed": false, "failed": true, "msg": "failed to retrieve file descriptor for interface: Transport endpoint is not connected"}

PLAY RECAP *********************************************************************
host0                      : ok=78   changed=37   unreachable=0    failed=1  
localhost                  : ok=6    changed=2    unreachable=0    failed=0   

looking at libvirt i see this:

Mar 22 18:54:22 tardis ovs-vsctl[6940]: ovs|00001|vsctl|INFO|Called as ovs-vsctl --timeout=5 -- --if-exists del-port vnet3
Mar 22 18:54:53 tardis ovs-vsctl[10138]: ovs|00001|vsctl|INFO|Called as ovs-vsctl --timeout=5 -- --if-exists del-port vnet4
Mar 22 19:04:16 tardis libvirtd[6526]: Failed to open file '/sys/class/net/tap0/operstate': No such file or directory
Mar 22 19:04:16 tardis libvirtd[6526]: unable to read: /sys/class/net/tap0/operstate: No such file or directory
Hint: Some lines were ellipsized, use -l to show in full.

**Comment by mbaldessari** [comment]
Worth checking https://bugzilla.redhat.com/show_bug.cgi?id=1198244

**Comment by larsks** [comment]
@mbaldessari: good find, although CentOS 7 has 1.2.17, which should be fixed.

@pkilambi: Can you verify the version of libvirt installed on your target box?  Also, by "consistently" do you mean "every time"? If that's the case I'd like to add some debugging around this particular point in the playbooks.

**Comment by pkilambi** [comment]
I already have the new version of libvirt and still have the problem. Interestingly i dont run into this when using instack-virt script. My undercloud comes up just fine. So its something in quickstart script thats throwing off libvirt.

Also note that this is a reinstall of quickstart. I had an undercloud install before on this box using quickstart which worked fine, then i had to reboot and when i try to rerun the quickstart it fails with the above. I tried manually removing all my old vms and re ran quickstart, and same thing. I guess some network element isnt cleaned up properly that quickstart uses?

For now I just used instack-virt script and everything is working. But quickstart does not work and fails with above error.
